tutorial/README.md

```diff
@@ -0,0 +1,824 @@
+
+```sh
+# You should see your normal username here (not 'azureuser')
+whoami 
+
+# You should see a cluster ending with '.azmk8s.io'
+# This should be the AKS cluster created by the Getting Started instructions 
+kubectl cluster-info
+
+kubectl get pods
+```
+
+You should see output like the following, this indicates that the application is running. 
+
+```txt
+NAME                          READY   STATUS    RESTARTS   AGE
+hello-world-548c548c8-nflnx   1/1     Running   0          6m20s
+```
+
+Next, let's look for a public IP address. Right now OAM will create a public endpoint for every Kubernetes service it creates.
+
+```sh
+kubectl get svc
+```
+
+If you see output like the following (`EXTERNAL-IP`), then your service has a public IP address assigned.
+
+```txt
+NAME          TYPE           CLUSTER-IP     EXTERNAL-IP     PORT(S)        AGE
+hello-world   LoadBalancer   10.0.195.163   52.247.202.29   80:31783/TCP   7m6s
+kubernetes    ClusterIP      10.0.0.1       <none>          443/TCP        2d
+```
+
+If the IP address is still in the pending state wait a few seconds and try again until the `EXTERNAL-IP` is populated.
+
+Now paste the IP address into your browser (your IP address will be different from the one shown here). After a few seconds you should see some basic info about the server.
+
+If this is working then you've successfully completed this step. You can now remove this application and move on the next step.
+
+```sh
+# using the control plane VM
+azoam hello-world/* | kubectl delete -f -
+```
+
+## Deploying Flight-Tracker
+
+Now you're ready to work with the flight tracker application, and write some of your own OAM definitions.
+
+The application being deployed is shown in the following diagram:
+
+![Application architecture diagram](service-tracker-diagram.jpg)
+
+---
+
+For this example we'll be running the PostgreSQL database in the cluster as a `ContainerizedWorkload`. This means that you'll need a total of 6 `Component` definitions and a single `ApplicationConfiguration`.
+
+You can find some starter manifests in `./flight-tracker` - the `quakes-api` service and database are done for you to introduce more of the concepts. You can divide all of these assets into individual files however you like. The tools don't enforce any restrictions. It's most convenient to place all of the files in a single directory so they can be applied using a directory wildcard.
+
+You can also find the completed sample in `./flight-tracker-solution` for reference if you want to check your answers.
+
+### 1. Create an empty ApplicationConfiguration
+
+Create an `ApplicationConfiguration` declaration with the name `flight-tracker`. Remember that an `ApplicationConfiguration` should use the resource type `ApplicationConfiguration@core.oam.dev/v1alpha2`. Feel free to refer to the previous example `./hello-world/applicationconfiguration.arm` as a template.
+
+For now the `spec.components` node should be an empty array.
```

Comment:
nit: For now,

Reply:
?

Reply:
Please don't leave feedback like `?`. What is the suggestion?

Reply:
It was to add a comma.